pkg/config/authorization/opa.go

```diff
@@ -81,3 +93,21 @@ func generatePolicyUID(policyName string, policyContent string, nonce int) strin
 	data := []byte(fmt.Sprint(nonce) + policyUIDHashSeparator + policyName + policyUIDHashSeparator + policyContent)
 	return fmt.Sprintf("%x", md5.Sum(data))
 }
+
+func downloadRegoDataFromUrl(url string) (string, error) {
+	if resp, err := http.Get(url); err != nil {
+		return "", fmt.Errorf("failed to fetch Rego config: %v", err)
+	} else {
+		defer resp.Body.Close()
+
+		body, err := ioutil.ReadAll(resp.Body)
```

Comment:
We need to specify very careful what external registries must comply with.

OPA's REST API, for example, returns an `application/json` to the Get Policy endpoint: https://www.openpolicyagent.org/docs/latest/rest-api/#get-a-policy. We don't have to support that specific data structure, but it is a reference to consider before we commit to any implementation.